aws_cloudfront_distribution.tf

```diff
@@ -14,7 +14,7 @@ resource "aws_cloudfront_distribution" "s3-distribution-TLS-v1" {
     }
   }
 
-  enabled             = true
+  enabled = true
 
   default_cache_behavior {
```

Comment:
**Accurics remediation**
 
**[LOW]: Ensure CloudFront Logging is enabled**
 
**Vulnerability**
Ensure that geo restriction is enabled for your Amazon CloudFront CDN distribution to whitelist or blacklist a country in order to allow or restrict users in specific locations from accessing web application content.
 
**Remediation**
With AWS CloudFront geo restriction you have the ability to block IP addresses based on Geo IP from reaching your CDN distribution and your web application content delivered by the distribution. The feature can also be used to assist in mitigation of Distributed Denial of Service (DDoS) attacks.

source: accurics-